PowerPointLabs/Test/FunctionalTest/CropToSameDimTest.cs

```diff
@@ -16,12 +16,10 @@ protected override string GetTestingSlideName()
         [TestCategory("FT")]
         public void FT_CropToSameTest()
         {
-            CropSameSizeSuccessfully();
             CropOneSideSuccessfully();
             CropTwoSidesSuccessfully();
             CropFourSidesSuccessfully();
             CropSmallerRefImgSuccessfully();
-            CropLargerRefImgSuccessfully();
```

Comment:
Ah they were moved to negative test cases, since cropping those cases will open a message box saying no copping was done